### PR TITLE
docs: add webpack 5 note on @project-serum/anchor

### PR DESCRIPTION
**Pull request by AlberErre**
Add documentation for `webpack 5` issue on frontend clients, details on #1290.

### CODE REVIEW

File: ts/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## @project-serum/anchor
```
Comment (armaniferrante) [inline ts/README.md:1]:
```suggestion
# @project-serum/anchor
```

File: ts/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# @project-serum/anchor
+
+TypeScript client for Anchor programs
```
Comment (armaniferrante) [inline ts/README.md:3]:
```suggestion
TypeScript client for Anchor programs
```

File: ts/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+[![npm](https://img.shields.io/npm/v/@project-serum/anchor.svg?color=blue)](https://www.npmjs.com/package/@project-serum/anchor)
+[![Docs](https://img.shields.io/badge/docs-typedoc-blue)](https://project-serum.github.io/anchor/ts/index.html) |
+
```
Comment (armaniferrante) [inline ts/README.md:7]:
```suggestion

TypeScript client for Anchor programs.

```

File: ts/README.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# @project-serum/anchor
+
+TypeScript client for Anchor programs
```
Comment (armaniferrante) [inline ts/README.md:3]:
```suggestion
```

File: ts/README.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# @project-serum/anchor
+
```
Comment (armaniferrante) [inline ts/README.md:2]:
```suggestion
```

File: ts/README.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+[![npm](https://img.shields.io/npm/v/@project-serum/anchor.svg?color=blue)](https://www.npmjs.com/package/@project-serum/anchor)
+[![Docs](https://img.shields.io/badge/docs-typedoc-blue)](https://project-serum.github.io/anchor/ts/index.html) |
```
Comment (armaniferrante) [inline ts/README.md:4]:
```suggestion
[![Docs](https://img.shields.io/badge/docs-typedoc-blue)](https://project-serum.github.io/anchor/ts/index.html)
```

**Comment by kise999** [comment]
So how to fix this error?

**Pull request merged**